frontend/src/turbo/turbo-event-listeners.ts

```diff
@@ -31,6 +31,7 @@ export function addTurboEventListeners() {
     const headers = event.detail.fetchOptions.headers as Record<string, string>;
     headers['Turbo-Referrer'] = window.location.href;
     headers['X-Turbo-Nonce'] = document.getElementsByName('csp-nonce')[0]?.getAttribute('content') || '';
+    headers['X-Authentication-Scheme'] = 'Session';
```

Comment:
I think you said in a meeting that you were surprised about the number of places where you had to add the header. I'd agree. What I am noticing though:

This place here looks different, it already looks rather generic (we hook into an event called `turbo:before-fetch-request`): Are the other places where we perform turbo requests not covered by this piece of code here? Or would this single change be enough?

Assuming that changing this place alone is not enough, it seems to me that most other places all reference `request` defined in `frontend/src/app/core/turbo/turbo-requests.service.ts`. This would mean that `request` would make for a good central place to define a header we truly want to attach to all our requests.

Reply:
I feared that. In this case I'd probably suggest to move some shared header handling into the `request` method of `turbo-requests.service.ts`, because that seems to be the service used by the other pieces of changed code.